test/conversions.jl

```diff
@@ -28,3 +28,27 @@ zdt_warsaw = ZonedDateTime(dt, warsaw; from_utc=true)
 # Identical since ZonedDateTime is immutable
 @test astimezone(zdt_utc, warsaw) === zdt_warsaw
 @test astimezone(zdt_warsaw, utc) === zdt_utc
+
+# ZonedDateTime to Unix timestamp (and vice versa)
+@test TimeZones.zdt2unix(ZonedDateTime(1970, utc)) == 0
+@test TimeZones.unix2zdt(0) == ZonedDateTime(1970, utc)
+
+no_dst = DateTime(2013, 2, 13)
+no_dst_zdt = ZonedDateTime(no_dst, warsaw)
+no_dst_offset = 3600
+@test TimeZones.zdt2unix(no_dst_zdt) == datetime2unix(no_dst) - no_dst_offset
```

Comment:
Good point. That modification makes the test pretty much exactly the function call. My concern is that doing a manual adjustment like this isn't a great practise. I'll try to think of an alternative way of doing this.